lib/pages/home/home.dart

```diff
@@ -339,8 +339,8 @@ class OBHomePageState extends ReceiveShareState<OBHomePage>
             overflow: Overflow.visible,
             children: <Widget>[
               const OBIcon(OBIcons.notifications),
-              _loggedInUserUnreadNotifications > 0
-                  ? Positioned(
+              _loggedInUserUnreadNotifications != null
```

Comment:
This is some other bug fix right?

Reply:
True, its unrelated to the feature but not a bug fix , my build was failing suddenly complaining this var was null, so I added a check.